packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -313,6 +313,7 @@ const PostsPagePostHeader = ({post, answers = [], dialogueResponses = [], toggle
             {commentCountNode}
           </LWTooltip> : commentCountNode}
           {isEAForum && <BookmarkButton post={post} className={classes.bookmarkButton} placement='bottom-start' />}
+          {isEAForum && <SharePostButton post={post} />}
```

Comment:
This is what it looks like for EAF:

<img width="635" alt="Screen Shot 2023-06-16 at 9 34 42 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/53994870-1197-47d7-a8b0-2a6abbb76ece">

See [figma](https://www.figma.com/file/d09iVzNSpZp9GyWGcfkWmX/Q2.2-2023?type=design&node-id=12-8961&t=ziHG7xr07rz9PLjF-0) for more on the design